extensions-contrib/opentelemetry-extensions/src/main/java/org/apache/druid/data/input/opentelemetry/protobuf/OpenTelemetryMetricsProtobufReader.java

```diff
@@ -143,25 +149,38 @@ private InputRow parseNumberDataPoint(NumberDataPoint dataPoint,
 
     if (dataPoint.hasAsInt()) {
       event.put(valueDimension, dataPoint.getAsInt());
-    } else if (dataPoint.hasAsDouble()) {
-      event.put(valueDimension, dataPoint.getAsDouble());
     } else {
-      throw new IllegalStateException("Unexpected dataPoint value type. Expected Int or Double");
+      event.put(valueDimension, dataPoint.getAsDouble());
     }
 
     event.putAll(resourceAttributes);
     dataPoint.getAttributesList().forEach(att -> event.put(metricAttributePrefix + att.getKey(),
-        getStringValue(att.getValue())));
+        parseAnyValue(att.getValue())));
 
     return createRow(TimeUnit.NANOSECONDS.toMillis(dataPoint.getTimeUnixNano()), event);
   }
 
-  private static String getStringValue(AnyValue value)
+  private static Object parseAnyValue(AnyValue value)
   {
-    if (value.getValueCase() == AnyValue.ValueCase.STRING_VALUE) {
-      return value.getStringValue();
+    switch (value.getValueCase()) {
+      case INT_VALUE:
+        return value.getIntValue();
+      case BOOL_VALUE:
+        return value.getBoolValue();
+      case ARRAY_VALUE:
+        return value.getArrayValue();
```

Comment:
how does this get treated?

Reply:
Does druid have a set of expected value types for columns? If it does, it would probably make sense for us to do some conversion from the disallowed types into strings or some other type.

Reply:
just fyi, for converter, we only support int, bool , double and string here: https://github.com/confluentinc/opencensus-protobuf-converter/blob/d96358a0d61f6f3524c3acc9d7db264a7a1a34c8/src/main/java/io/confluent/connect/opentelemetry/OpenTelemetryConverter.java#L91-L93 Not sure whether we want to keep it consistent here.

Reply:
It looks like the OpenCensusProtobufProtobufReader changes all values into strings. similar to what we do in the converter, so I'll do the same thing here.

Reply:
the link i provided was for attributes, not metrics value. This should be the correct one: https://github.com/confluentinc/opencensus-protobuf-converter/blob/d96358a0d61f6f3524c3acc9d7db264a7a1a34c8/src/main/java/io/confluent/connect/opentelemetry/OpenTelemetryMetricsConverter.java#L121-L166